CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Change Log
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) and adheres to [Semantic Versioning](http://semver.org) as of version 1.0.0.
```

Comment:
As far as I know, we don't commit to SemVer in any other projects (at least not any of the Blue ones).